.travis.yml

```diff
@@ -137,6 +137,22 @@ matrix:
 
         - ./ci/travis/test-wheels.sh
 
+    # Test RLlib.
+    - os: linux
+      env:
+        - PYTHONWARNINGS=ignore
+      install:
+        - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
+        - if [ RAY_CI_RLLIB_AFFECTED != "1" ]; then exit; fi
+        - ./ci/travis/install-bazel.sh
+        - ./ci/suppress_output ./ci/travis/install-dependencies.sh
+        - export PATH="$HOME/miniconda/bin:$PATH"
+        - ./ci/suppress_output ./ci/travis/install-ray.sh
+      script:
+        - if [ $RAY_CI_RLLIB_AFFECTED != "1" ]; then exit; fi
+        - ./ci/suppress_output bazel test --build_tests_only --show_progress_rate_limit=100 --test_output=errors rllib/...
```

Comment:
You probably want to copy the "bazel python tests" command that runs the other python ray tests below, instead of the cc one (it has more comprehensive set of flags).

Reply:
ok, will change ...